sycl/test-e2e/Basic/max_linear_work_group_size_props.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %{build} -o %t.out
+// RUN: %{build} -o %t.out -Wno-deprecated-declarations
```

Comment:
Unless we specifically want to test the old API, we shouldn't add this flag but rewrite the test instead. The reason is fear that we can accidentally use even more deprecated APIs in a test, thus making even more work for ourselves in the future PR which removes deprecated overloads.